addons/editor-compact/userstyle.css

```
@@ -310,6 +310,16 @@ body:not(.sa-columns-enabled) [class*="gui_tab-panel"] [class*="gui_extension-bu
 [class*="selector_list-area"] > div:first-child {
   margin-top: 0.25rem;
 }
+[class*="sprite-selector-item_number"] {
+  margin: 0 -2px;
+  padding: 0 2px;
+  background-color: var(--editorDarkMode-selector2);
```

There isn't a fallback for when `editor-dark-mode` is disabled.
```suggestion
  background-color: var(--editorDarkMode-selector2, #d9e3f2);
```